modules/cco-ccoctl-configuring.adoc

```diff
@@ -111,6 +111,28 @@ ifdef::aws-sts[]
 * `s3:PutObjectAcl`
 * `s3:PutObjectTagging`
 
+|====
++
+If you plan to store the OIDC configuration in a private S3 bucket that is accessed by the IAM identity provider through a public CloudFront distribution URL, the AWS account requires the following additional permissions:
++
+.Additional permissions for a private S3 bucket with CloudFront
+[cols="a,a"]
+|====
+|`cloudFrontClient` permissions |`s3Client` permissions
+
+|* `cloudFrontClient.CreateCloudFrontOriginAccessIdentity`
+* `cloudFrontClient.DeleteCloudFrontOriginAccessIdentity`
+* `cloudFrontClient.GetCloudFrontOriginAccessIdentity`
+* `cloudFrontClient.ListCloudFrontOriginAccessIdentities`
+* `cloudFrontClient.CreateDistributionWithTags`
+* `cloudFrontClient.DeleteDistribution`
+* `cloudFrontClient.GetDistribution`
+* `cloudFrontClient.UpdateDistribution`
+* `cloudFrontClient.ListDistributions`
+* `cloudFrontClient.ListTagsForResource`
+|* `s3Client.PutPublicAccessBlock`
+* `s3Client.PutBucketPolicy`
```

Comment:
Translating to IAM Policy:
```suggestion
|* `cloudfront:GetCloudFrontOriginAccessIdentityConfig`
* `cloudfront:ListCloudFrontOriginAccessIdentities`
* `cloudfront:ListTagsForResource`
* `cloudfront:DeleteCloudFrontOriginAccessIdentity`
* `cloudfront:ListDistributions`
* `cloudfront:GetCloudFrontOriginAccessIdentity`
* `cloudfront:UpdateDistribution`
* `cloudfront:CreateDistribution`
* `cloudfront:CreateCloudFrontOriginAccessIdentity`
|* `s3:PutBucketPolicy`
* `s3:PutBucketPublicAccessBlock`
```

I checked item by item from that list, comparing with the PR introducing CloudFront https://github.com/openshift/cloud-credential-operator/pull/486 and the recent document [describing manual steps to migrate from public to private](https://github.com/openshift/cloud-credential-operator/blob/master/docs/sts-migrate-to-private-bucket.md#prerequisites-). The suggested list seems to be accurate with all current listings.

Now we need to make sure if is there any other action not captures by those sources.